src/external/nanopb.BUILD

```diff
@@ -4,6 +4,10 @@ exports_files(["LICENSE.txt"])
 
 package(default_visibility = ["//visibility:public"])
 
+common_defines = [
+    "PB_FIELD_16BIT",
```

Comment:
comment for why we have this would be helpful

Reply:
added
```
    # By default, NanoPb only supports 8-bit tags. This changes the tag type to
    # one that supports 16-bit tags.
    # (https://jpa.kapsi.fi/nanopb/docs/reference.html)
```

Reply:
But _why_ do we need 16-bit tags? Why aren't 8-bit tags sufficient?

Reply:
So I'm actually not sure why we can't get away with 8-bit tags, even after digging at it for a bit. I could spend more time and probably figure it out, but not sure that's worth it here?

Reply:
@akhilveeraghanta any idea why? Are there too many fields to tag with 8 bits?

Reply:
The funky thing is that there def. shouldn't be anything with more then 255 fields. It probably has something to with the fixed sized repeated field in the `PrimitiveSetMsg`, but still not quite sure why..........

Reply:
~~~
#if !defined(PB_FIELD_16BIT) && !defined(PB_FIELD_32BIT)                                                                                                                                                                                                                  
     /* If you get an error here, it means that you need to define PB_FIELD_16BIT                                                                                                                                                                                              
      * compile-time option. You can do that in pb.h or on compiler command line.                                                                                                                                                                                              
      *                                                                                                                                                                                                                                                                        
      * The reason you need to do this is that some of your messages contain tag                                                                                                                                                                                               
      * numbers or field sizes that are larger than what can fit in the default                                                                                                                                                                                                
      * 8 bit descriptors.                                                                                                                                                                                                                                                     
     */        
~~~

From the generated PrimitiveSetMsg

Nanopb determined that 8bits is not sufficient, we just have to roll with it